server/src/dtos/asset-response.dto.ts

```diff
@@ -131,7 +131,9 @@ export function mapAsset(entity: AssetEntity, options: AssetMapOptions = {}): As
 }
 
 export class MemoryLaneResponseDto {
+  @ApiProperty({ deprecated: true })
   title!: string;
```

Comment:
My initial thought for this property is that we will have more categories, not just `x year ago`, but perhaps the `best shots of the month`...etc.

Reply:
It doesn't work very well with translations though. I think sending back text in an API always gets complicated when dealing with i18n. We can probably add a type field if we start supporting another category of memories.

Reply:
Basically metadata and types are probably the easy to go and the clients can translate it into titles